quickwit-cli/src/cli.yaml

```diff
@@ -196,8 +196,42 @@ subcommands:
         settings:
             - ArgRequiredElseHelp
         display_order: 2
-        about: Operations (add, delete, describe...) on splits.
+        about: Operations (list, add, delete, describe...) on splits.
         subcommands:
+            - list:
+                about: List all splits of an index.
+                args:
+                    - metastore-uri:
+                        about: Location of the metastore
+                        long: metastore-uri
+                        value_name: METASTORE URI
+                        required: true
+                        env: QUICKWIT_METASTORE_URI
+                    - index-id:
+                        about: Index ID
+                        long: index-id
+                        value_name: INDEX ID
+                        required: true
+                    - states:
+                        about: Comma-separated list of SplitState to filter on.
+                        long: states
+                        value_name: SPLIT STATES
+                        multiple_occurrences: true
+                        use_delimiter: true
+                    - from:
+                        about: Start date to filter on.
+                        long: from
+                        value_name: START TIMESTAMP
+                    - to:
+                        about: End date to filter on.
+                        long: to
+                        value_name: END TIMESTAMP
+                    - tags:
+                        about: Comma-separated list of tags, only splits that contain all of the tags will be returned.
```

Comment:
I don't know about this behavior. Anybody, thoughts?

Reply:
https://github.com/quickwit-inc/quickwit/pull/841/#discussion_r767541011